doc/changelog/11-standard-library/18936-remove-ByteVector-Ndist-Ndigits.rst

```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  The library files ``Coq.NArith.Ndigits``, ``Coq.NArith.Ndist``, and ``Coq.Strings.ByteVector``
```

Comment:
maybe
```suggestion
  The library files ``Coq.NArith.Ndigits``, ``Coq.NArith.Ndist``, and ``Coq.Strings.ByteVector``
  which were deprecated since 8.19
```